widget_driver/example/test/widget_tests/playground/playground_test_widget/playground_test_widget_test.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:example/widgets/playground/playground_test_widget/playground_test_child_widget.dart';
+import 'package:example/widgets/playground/playground_test_widget/playground_test_child_widget_driver.dart';
+import 'package:example/widgets/playground/playground_test_widget/playground_test_widget.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:widget_driver/widget_driver.dart';
+
+void main() {
+  group('PlaygroundTestWidget:', () {
+    testWidgets('Some test where we make sure the PlaygroundTestChildWidget uses real driver',
```

Comment:
This here is the test which show cases how to use the new `WidgetDriverTestConfigProvider`